pkg/metrics/metrics.go

```diff
@@ -61,12 +62,22 @@ var CertificateExpiryTimeSeconds = prometheus.NewGaugeVec(
 	[]string{"name", "namespace"},
 )
 
+var CertificateIssuedLatency = prometheus.NewHistogram(
+	prometheus.HistogramOpts{
+		Namespace: namespace,
+		Name:      "certificate_issued_latency",
+		Buckets:   []float64{30, 60, 120, 180, 240, 300},
+		Help:      "The amount of time for a namespace secret to be issued after a certificate has been created.",
```

Comment:
> for a namespace secret 

I'm not sure what 'namespace secret' means here vs a 'non-namespace' secret? Can you update the Help text to be clearer? 😄